util/image_viewer/slip_viewer/tile_generation.py

```diff
@@ -296,14 +296,28 @@ def set_image(
         if not isinstance(image_data, tuple):
             image_data = (image_data,)
 
-        self.flex_image = get_flex_image_multipanel(
-            brightness=self.current_brightness / 100,
-            panels=detector,
-            show_untrusted=self.show_untrusted,
-            image_data=image_data,
-            beam=self.raw_image.get_beam(),
-            color_scheme=self.current_color_scheme,
-        )
+        if len(detector) > 1:
```

Comment:
@benjaminhwilliams it appears that it is this change that causes the threshold pixel failure. That is, if I change this to `if True` then the failure does not occur

Reply:
It seems that the return types of `get_flex_image_multipanel` and `get_flex_image` are dissimilar.

Reply:
Yes, `get_flex_image`→ `FlexImage` whereas `get_flex_image_multipanel`→ `generic_flex_image` :thinking:

Reply:
Yes, I've discovered the same in parallel. Without more documentation, I'm struggling to understand what each type is meant to represent (there are three, I think — FlexImage, FlexImage_d and generic_flex_image, which is a class in spite of the lack of camel case).